scripts/question_answering/run_squad.py

```diff
@@ -491,7 +493,7 @@ def train(args):
 
     logging.info('Creating distributed trainer...')
     # Collect differentiable parameters
-    param_dict = qa_net.collect_params()
+    param_dict = deduplicate_param_dict(qa_net.collect_params())
```

Comment:
does the result make more sense after correcting for duplication?

Reply:
The results are better I think. The main reason is that it fixed the gradient norm calculation. Also, it does not have an impact on the MXNet part because it has been deduplicated already in MXNet (as confirmed with @leezu ).